libcxx/test/libcxx/atomics/atomics.types.generic/atomics.types.float/lockfree.pass.cpp

```diff
@@ -45,7 +45,8 @@ void test() {
 int main(int, char**) {
   test<float>();
   test<double>();
-  test<long double>();
+  // https://github.com/llvm/llvm-project/issues/47978
```

Comment:
```suggestion
  // TODO https://github.com/llvm/llvm-project/issues/47978
```
This makes it easier to find them.